platform-tests/src/test/java/org/junit/AutomaticModuleNameTests.java

```diff
@@ -59,6 +59,7 @@ private static String version(String module) {
 		throw new AssertionError("module name is unknown: " + module);
 	}
 
+	@SuppressWarnings("FilesLinesLeak") // stream closed indirectly by JUnit 5
```

Comment:
It should be wrapped in a `try-resource` block anyway. Like was done below with `try (JarFile jarFile = new JarFile(jarPath.toFile())) { ...}`.